### ISSUE TITLE
Image Printing 

### ISSUE THREAD

**Issue by ahsan-ab** [dikoga/StarIOPrinterBinding#1]
Hi,

Currently i'm working with "Star Micronics" Printer TCP/BT/USB & successfully printer text(only English want to print Arabic :| ) but i want to print image have you ever print image ?

If yes let me know how?

**Comment by dikoga** [comment]
No, I have not. Probably there is a command that lets the printer understand it is an image and prints it.
